scripts/prefect_scripts/.env.example

```diff
@@ -0,0 +1,28 @@
+# Example Environment Configuration for Ax Tutorial
+# Copy this file to .env or set these environment variables
+
+# MongoDB Configuration
+# For local MongoDB installation:
+# MONGODB_CONNECTION_STRING=mongodb://localhost:27017/
+
+# For MongoDB Atlas (cloud):
+# MONGODB_CONNECTION_STRING=mongodb+srv://username:password@cluster.mongodb.net/
+
+# Database settings
+MONGODB_DATABASE=ax_tutorial
+MONGODB_COLLECTION=optimization_checkpoints
+
+# Prefect Configuration
+# For Prefect Cloud, set your API key:
+# PREFECT_API_KEY=your_api_key_here
+
+# For local Prefect server:
+# PREFECT_API_URL=http://127.0.0.1:4200/api
+
+# Optional: Slack notifications
+# SLACK_WEBHOOK_URL=https://hooks.slack.com/services/YOUR/WEBHOOK/URL
```

Comment:
Doesn't this get set up once and on Prefect cloud? Why would you need a separate env  var for it?